tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.csproj

```diff
@@ -19,9 +23,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="public_privatekey.pfx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.Core" Version="3.3.106.22" />
     <PackageReference Include="Dapper" Version="2.0.35" />
     <PackageReference Include="FluentAssertions" Version="5.10.2" />
     <PackageReference Include="Flurl.Http" Version="2.4.2" />
+    <PackageReference Include="jose-jwt" Version="2.5.0" />
```

Comment:
Is this used in order to create the JSON key? Just a brief description used as a reference later on.

Reply:
Ah, no, it snuck in from when I was prototyping various solutions. I'll remove it